tests/routes/resetPassword.test.js

```diff
@@ -0,0 +1,154 @@
+import chai, {
+  expect
+} from 'chai';
+import server from '../../index';
+import config from '../../config';
+import UserHelper from '../../helpers/UserHelper';
+import TokenHelper from '../../helpers/TokenHelper';
+
+const user = {
+  email: 'resettester@havenmail.com',
+  username: 'resettester',
+  password: 'testedokay',
+};
+const beginResetEndpoint = '/api/v1/users/reset-password/begin';
```

Comment:
@Alameen688 . I like the organization of your code. I was wondering though. Why do you have two endpoints here?